src/Symfony/Component/Messenger/Bridge/AmazonSqs/Middleware/WithMessageDeduplicationId.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\AmazonSqs\Middleware;
+
+/**
+ * @see https://docs.aws.amazon.com/en_gb/AWSSimpleQueueService/latest/SQSDeveloperGuide/using-messagededuplicationid-property.html
+ */
+interface WithMessageDeduplicationId
```

Comment:
interfaces *must* end with the `Interface` suffix

Reply:
Damn ! I'd totally forgotten about that rule.

Is `WithMessageDeduplicationIdInterface` sound ok ?

Or should I rename with more traditionnal approch like `SupportMessageDeduplicationInterface`may be ?